engine/src/test/java/org/destinationsol/IniReaderTest.java

```diff
@@ -51,7 +51,8 @@ public void initIniReader() {
                         "floatKey = 6\n" +
                         "anotherFloatKey = 7.3f\n" +
                         "invalidFloatKey = 8,6\n" +
-                        "anotherInvalidFloatKey = hi\n"; // Each of these values is used exactly once in these tests, except for doubleRequestedKey
+                        "anotherInvalidFloatKey = hi\n" +
+                        "UnicodeKey çáč🧝 = unicodevalue áśǵj́ḱĺóí⋄«»⋄⋄ǫő"; // Each of these values is used exactly once in these tests, except for doubleRequestedKey
```

Comment:
It would probably help to move this comment to the very top of the list. That way it doesn't wind up being caught in diffs like this (also, seems like it would get read sooner at the top anyway)